задание6/Task61.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace Задание6.1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var m = int.Parse(Console.ReadLine());
+            var n = int.Parse(Console.ReadLine());
+            static bool НазваниеМетода(int m, int n)
+            {
+                return ((m + n) % 2 == 0) && ((m * n) % 3 == 0);
+            }
+            Console.WriteLine(НазваниеМетода(m, n));
+            Console.ReadKey();
+        }
+    }
```

Comment:
Старайтесь без особых причин не использовать метод в методе. И название нормальное можно было бы придумать.
```suggestion
    class Program
    {
        static void Main(string[] args)
        {
            var m = int.Parse(Console.ReadLine());
            var n = int.Parse(Console.ReadLine());

            Console.WriteLine(НазваниеМетода(m, n));
            Console.ReadKey();
        }
        
        static bool НазваниеМетода(int m, int n)
        {
           return ((m + n) % 2 == 0) && ((m * n) % 3 == 0);
        }
    }
```